.editorconfig

```diff
@@ -0,0 +1,20 @@
+# https://editorconfig.org/
+
+# Inherit rules for everything else
+root = false
+
+# Default rules for all files
+[*]
+charset                  = utf-8
+insert_final_newline     = true
+trim_trailing_whitespace = true
+
+# Go
+[*.go]
+indent_style = tab
+indent_size  = 4
```

Comment:
I think this may not match gofmt, which uses width 8. Can we remove the Go section from this file, and we can rely on gofmt for formatting Go code?

Reply:
The tab width doesn't actually affect the files, just how it is displayed in the online editor in GitHub, in Vim, and VS Code, etc. I can change the display width to 8 if you prefer? But either way this won't interfere with `go fmt -s`.

Reply:
Thank you. I did not know that. In this case, we can keep it.